package.json

```diff
@@ -1,31 +1,17 @@
 {
-  "name": "html2pdf.js",
-  "version": "0.9.0",
+  "name": "htmlpdfconverter",
+  "version": "1.0.2",
```

Comment:
What's going on here, and around this whole PR? just wondering :-)

Reply:
These changes were generated through an automated configuration. Have reverted all of them. Sorry for the inconvenience. Have raised a separate PR https://github.com/eKoopmans/html2pdf/pull/164 mentioning only the required changes and to the correct target branch